src/main/java/io/reactivex/internal/operators/observable/ObservableBufferExactBoundary.java

```diff
@@ -17,6 +17,7 @@
 import java.util.concurrent.Callable;
 
 import io.reactivex.*;
+import io.reactivex.ObservableSource;
```

Comment:
There's superfluous imports like this throughout this PR and your other one, but I don't think it's a big deal.